uptane-standard.md

```diff
@@ -714,7 +723,7 @@ There may be several different filenames that all refer to the same image binary
 
 #### Send latest time to secondaries {#send_time_primary}
 
-The primary SHOULD send the time to each ECU. The secondary will verify the time message, then overwrite its current time with the received time. The primary MAY omit this step if the secondary has its own way of loading and verifying the time.
+The primary SHALL send the time to each ECU. The secondary will verify the time message, then overwrite its current time with the received time. The primary MAY omit this step if the secondary has its own way of loading and verifying the time.
```

Comment:
Should not be `SHALL` as some ECUs cannot support checks like that. `SHOULD` is appropriate here.

Reply:
OK that is fair. 

However, if it is optional then implementer's _may_ not send time to secondary ECUs which **are** capable of these checks.
I will rewrite the MAY sentence to indicate this step may be omitted for secondaries not capable of performing this check.